### PR TITLE
seeded and connected front end

### PR DESCRIPTION
**Pull request by tak074**

### CODE REVIEW

File: Client/App.jsx [hunk 1]
```diff
@@ -68,16 +69,33 @@ class ImageCarousel extends React.Component {
...
     if(window.location.pathname.slice(1) === '') {
-      productID = 1;
-    } else if (window.location.pathname.slice(1) >= 101) {
-      productID = 1;
+      productID = 0;
+    } else if (window.location.pathname.slice() >= 100) {
+      productID = 0;
     } else {
       productID = window.location.pathname.slice(1);
     }
```
Comment (mpstanford) [inline Client/App.jsx:80]: I think line 76 is missing a 1 in the parentheses. Without it, you'll be including the question mark, so the expression will be something like:

else if ('?102' >= 100)

File: Server/server.js [hunk 2]
```diff
@@ -1,72 +1,170 @@
...
-  var alt = req.body.alt;
-
-  Image.create({product, imageName, color, url, alt}, function(err, result) {
+  let image_name = req.body.image_name;
+  let color = req.body.color;
+  let url = req.body.url;
+  let category = req.body.category;
+  let picture_length = req.body.picture_length;
+  let alt = req.body.alt;
+  let product;
```
Comment (mpstanford) [inline Server/server.js:137]: could use destructuring to make this a little cleaner: 

`let {image_name, color, url, category, picture_length, alt} = req.body`

**Review by mpstanford — APPROVED** [review]
Looks great, TK! It was hard to find things to comment on.

File: Client/App.jsx (hunk 1 above)
Comment (tak074) [reply Client/App.jsx:80]: great catch! thank you!! I have edited this now

File: Server/server.js (hunk 2 above)
Comment (tak074) [reply Server/server.js:137]: that is definitely much cleaner. I will use that!

**Pull request merged**